Tasks/SqlAzureDacpacDeploymentV1/SqlAzureActions.ps1

```diff
@@ -264,10 +264,10 @@ function Run-SqlCmd {
         $sqlUsername = Get-FormattedSqlUsername -sqlUserName $sqlUsername -serverName $serverName
     }
 
-    $scriptArgument = "Invoke-Sqlcmd -ServerInstance `"$serverName`" -Database `"$databaseName`" -Username `"$sqlUsername`" "
+    $scriptArgument = "Invoke-Sqlcmd -ServerInstance `"$serverName`" -Database `"$databaseName`" -Username '$sqlUsername' "
 
-    $commandToRun = $scriptArgument + " -Password `"$sqlPassword`" "
-    $commandToLog = $scriptArgument + " -Password ****** "
+    $commandToRun = $scriptArgument + " -Password '$sqlPassword' "
```

Comment:
- Should we define password as \`'$sqlPassword\`' instead of '$sqlPassword' as I think with only single quotes variable won't be expanded.
- It will fail when the password itself contains the single quotes. 
- Try with password:      ~`!@#$%^&*()_+{}[]:;<>,.?/Aa"'